controller/api/apis.go

```diff
@@ -2882,7 +2882,14 @@ type RESTCrdDlpGroupConfig struct {
 }
 
 type RESTDlpSensorExport struct {
-	Names []string `json:"names"`
+	Names         []string           `json:"names"`
+	ExportOptions *RESTExportOptions `json:"export_options"`
+}
+
+type RESTExportOptions struct {
+	Type            string `json:"type"`
```

Comment:
What is "type" field for? Is it for "git service type"? If yes, why it's in RESTExportOptions but not in REST_RemoteExportRepository_POST?